test/explorer/data_frame_test.exs

```diff
@@ -560,7 +560,7 @@ defmodule Explorer.DataFrameTest do
                "calc2" => :integer,
                "calc3" => :integer,
                "calc4" => :float,
-               #  "calc5" => :integer,
+               "calc5" => :integer,
```

Comment:
Is this correct? The results above are float, no?

Reply:
It's a bug #374.

dtype of series is float, but dtype of dataframe column is integer.

Reply:
So maybe we should leave this one commented and add a test where the first argument is a float?